nxdrive/metrics/constants.py

```diff
@@ -11,17 +12,24 @@
 UPDATER_CHANNEL = "updater.channel"
 METRICS_LOCALE = "locale"
 
-CONFLICT_HIT = "directEdit.conflict.hit"
-RECOVERY_HIT = "directEdit.recovery.hit"
+# Direct Edit
+DE_CONFLICT_HIT = "directEdit.conflict.hit"
+DE_RECOVERY_HIT = "directEdit.recovery.hit"
 DE_ERROR_COUNT = "directEdit.error.count"
 DE_SAVE_COUNT = "directEdit.save.count"
 
-SESSION_NUMBER = "directTransfer.session.number"
-DUPLICATE_BEHAVIOUR = "directTransfer.duplicate.file.behavior"
-NEW_FOLDER = "directTransfer.option.newFolder"
-SESSION_STATUS = "directTransfer.session.status"
+# Direct Transfer
+DT_SESSION_NUMBER = "directTransfer.session.number"
+DT_DUPLICATE_BEHAVIOUR = "directTransfer.duplicate.file.behavior"
```

Comment:
```suggestion
DT_DUPLICATE_BEHAVIOR = "directTransfer.duplicate.file.behavior"
```

Reply:
People in UK will hate us for this but alright.

Reply:
Behvior is uniformized across the whole source code ;)